GVFS/GVFS.UnitTests/Prefetch/DiffHelperTests.cs

```diff
@@ -44,12 +65,17 @@ public class DiffHelperTests
         public void CanParseDiffForwards()
         {
             MockTracer tracer = new MockTracer();
-            DiffHelper diffForwards = new DiffHelper(tracer, new MockGVFSEnlistment(), new List<string>(), new List<string>());
+            DiffHelper diffForwards = new DiffHelper(tracer, new MockGVFSEnlistment(), new List<string>(), new List<string>(), includeSymLinks: includeSymLinks);
             diffForwards.ParseDiffFile(GetDataPath("forward.txt"), "xx:\\fakeRepo");
 
-            // File added, file edited, file renamed, folder => file, edit-rename file
+            // File added, file edited, file renamed, folder => file, edit-rename file, SymLink added
             // Children of: Add folder, Renamed folder, edited folder, file => folder
-            diffForwards.RequiredBlobs.Count.ShouldEqual(9);
+            diffForwards.RequiredBlobs.Count.ShouldEqual(10);
+
+            if (diffForwards.shouldIncludeSymLinks)
+            {
+                diffForwards.FileAddOperations.ContainsKey("symLinkToBeCreated.txt");
+            }

```

Comment:
I don't think this check is validating anything as it is right now (there is nothing checking the return value of `ContainsKey`.

Maybe replace this if block with:

```
diffForwards.FileAddOperations.ContainsKey("symLinkToBeCreated.txt").ShouldEqual(diffForwards.shouldIncludeSymLinks);
```